### PR TITLE
Added useful timestamps.

### PR DESCRIPTION
**Pull request by zond**
Added Game.StartedAt and Phase.CreatedAt, and an endpoint to backfill correct timestamps for those fields.

Also changed all 401 error messages to "unauthenticated" instead of the incorrect "unauthorized".

### CODE REVIEW

**Comment by zond** [comment]
Tom, since this is a combination "add new field" and "backfill new fields via administrative endpoint" change, it would be nice to have a pair of extra eyes.

Also, sorry for mixing it up with the change in wording for 401 errors, it should probably have been in a patch of its own.

**Comment by zond** [comment]
This is for #104

**Comment by zond** [comment]
Added some dry-run logging functionality, will merge and test it in dry run mode.

File: game/handler.go [hunk 1]
```diff
@@ -435,8 +442,86 @@ func handleConfigure(w ResponseWriter, r Request) error {
...
+						if phase.PhaseOrdinal != int64(i) {
+							return fmt.Errorf("WTF, the phases aren't sorted properly? Phase %v is %+v", i, phase)
+						}
+						if phase.CreatedAt.IsZero() {
+							// If this is the first phase OR DeadlineAt - lastPhase.CreatedAt > phase length.
+							if i == 0 || phase.DeadlineAt.Sub(phases[i-1].CreatedAt) > time.Minute*game.PhaseLengthMinutes {
+								phase.CreatedAt = phase.DeadlineAt.Add(-time.Minute * game.PhaseLengthMinutes)
+								log.Infof(ctx, "Updating phase of game with phase length %v, with DeadlineAt %v and no previous phase within %v to have CreatedAt %v", time.Minute*game.PhaseLengthMinutes, phase.DeadlineAt, time.Minute*game.PhaseLengthMinutes, phase.CreatedAt)
+							} else {
+								phase.CreatedAt = phase.DeadlineAt
```
Comment (tttppp) [inline game/handler.go]: I don't understand this. Is this the case when there are no actions needed by players (e.g. Spring 1901 retreat).

Comment (zond) [reply game/handler.go]: Exactly.

If all players are RDY - which happens if they have no Options, NMRed last phase, or are eliminated - the phase resolved immediately and gets a DeadlineAt when created.

**Pull request merged**